.github/workflows/release.yml

```
@@ -0,0 +1,38 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - feat_v3.x.
```

actionlint (1.7.4)

character '.' is invalid for branch and tag names. ref name must not end with / and .. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)